internal/http/services/owncloud/ocdav/put.go

```diff
@@ -307,6 +307,8 @@ func (s *svc) handlePut(ctx context.Context, w http.ResponseWriter, r *http.Requ
 		}
 		Propagator.Inject(ctx, propagation.HeaderCarrier(httpReq.Header))
 		httpReq.Header.Set(datagateway.TokenTransportHeader, token)
+		httpReq.Header.Set(net.HeaderUploadLength, strconv.FormatInt(length, 10))
```

Comment:
I think the httpReq.Header.Set(net.HeaderUploadLength line is unnecessary as we are using the `simple` PUT, not tus.
```suggestion
```